modules/@apostrophecms/image/ui/apos/components/AposMediaUploader.vue

```diff
@@ -82,6 +82,11 @@ export default {
     },
     async uploadMedia (event) {
       try {
+        apos.bus.$emit('apos-busy', {
```

Comment:
With `busy: true` on the http call I would be surprised if you were spending significant time not already marked busy for you by that.